eva-accession-release/src/test/java/uk/ac/ebi/eva/accession/release/steps/processors/ExcludeInvalidVariantsProcessorTest.java

```diff
@@ -0,0 +1,148 @@
+        IVariant variant = newVariant(VALID_ALLELE, ALL_VALID_LETTERS_LOWER_CASE_ALLELE);
+        assertEquals(variant, processor.process(variant));
+    }
+
+    @Test
+    public void referenceAlleleLowerAndUpperCaseValid() throws Exception {
+        IVariant variant = newVariant(ALL_VALID_LETTERS_LOWER_AND_UPPER_CASE_ALLELE, VALID_ALLELE);
+        assertEquals(variant, processor.process(variant));
+    }
+
+    @Test
+    public void alternateAlleleLowerAndUpperCaseValid() throws Exception {
+        IVariant variant = newVariant(VALID_ALLELE, ALL_VALID_LETTERS_LOWER_AND_UPPER_CASE_ALLELE);
+        assertEquals(variant, processor.process(variant));
+    }
+
+    @Test
+    public void referenceAlleleInvalidLetters() throws Exception {
+        IVariant variant = newVariant(INVALID_LETTERS_ALLELE, VALID_ALLELE);
+        assertNull(processor.process(variant));
+    }
+
+    @Test
+    public void alternateAlleleInvalidLetters() throws Exception {
+        IVariant variant = newVariant(VALID_ALLELE, INVALID_LETTERS_ALLELE);
+        assertNull(processor.process(variant));
+    }
+
+    @Test
+    public void referenceAlleleSpecialCharacters() throws Exception {
+        IVariant variant = newVariant(SPECIAL_CHARACTERS_ALLELE, VALID_ALLELE);
+        assertNull(processor.process(variant));
+    }
+
+    @Test
+    public void alternateAlleleSpecialCharacters() throws Exception {
+        IVariant variant = newVariant(VALID_ALLELE, SPECIAL_CHARACTERS_ALLELE);
+        assertNull(processor.process(variant));
+    }
+
+    @Test
+    public void referenceAlleleNamed() throws Exception {
+        IVariant variant = newVariant(NAMED_ALLELE, VALID_ALLELE);
+        assertNull(processor.process(variant));
+    }
+
+    @Test
+    public void alternateAlleleNamed() throws Exception {
+        IVariant variant = newVariant(VALID_ALLELE, NAMED_ALLELE);
+        assertNull(processor.process(variant));
+    }
+
+    @Test
+    public void referenceAlleleSpaces() throws Exception {
+        IVariant variant = newVariant(SPACES_ALLELE, VALID_ALLELE);
+        assertNull(processor.process(variant));
+    }
+
+    @Test
+    public void alternateAlleleSpaces() throws Exception {
+        IVariant variant = newVariant(VALID_ALLELE, SPACES_ALLELE);
+        assertNull(processor.process(variant));
+    }
+
+    @Test
+    public void referenceAlleleEmpty() throws Exception {
+        IVariant variant = newVariant(EMPTY_ALLELE, VALID_ALLELE);
+        thrown.expect(IllegalArgumentException.class);
+        thrown.expectMessage(REFERENCE_AND_ALTERNATE_ALLELES_CANNOT_BE_EMPTY);
+        assertNull(processor.process(variant));
+    }
+
+    @Test
+    public void alternateAlleleEmpty() throws Exception {
+        IVariant variant = newVariant(VALID_ALLELE, EMPTY_ALLELE);
+        thrown.expect(IllegalArgumentException.class);
+        thrown.expectMessage(REFERENCE_AND_ALTERNATE_ALLELES_CANNOT_BE_EMPTY);
+        assertNull(processor.process(variant));
```

Comment:
a small detail: if `process` is going to throw an exception, the `assertNull` is misleading and unused.

here and in the previous test